crates/jit/src/instantiate.rs

```diff
@@ -103,6 +105,22 @@ impl<'data> RawCompiledModule<'data> {
         // Make all code compiled thus far executable.
         compiler.publish_compiled_code();
 
+        // Initialize profiler and load the wasm module
+        match profiler {
+            Some(_) => {
+                let region_name = String::from("wasm_module");
+                let mut profiler_new = profiler.unwrap().lock().unwrap();
```

Comment:
`profiler_new` -> `profiler`

Reply:
@yurydelendik Shadowing the profiler variable name acceptable here?

Reply:
yes

Reply:
Should be done. Last push removed a comment.